source/blog/articles/2017-12-cmo-is-back.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+
+title: "CMO is Back!"
+date: "2017-12-09"
+author: ["Joshua Steele", "joshukraine", "@tw"]
+image: "blog/2017/cmo-is-back-1200w.jpg"
+caption: "Today we're excited to announce that CMO is coming back in the summer of 2018! Read on to get the details and learn how to apply."
+
+---
+
+Carpathian Mountain Outreach 2018 will be our eleventh project — a few weeks of intense evangelism and hands-on missions training. To date, more than 40 men have trekked with our team through the mountains of western Ukraine, bringing the Gospel to multitudes. Listen to what they've been saying...READMORE
```

Comment:
Is the "READMORE" thing supposed to be there?

Reply:
Yes. It doesn't have to be on a new line.